.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: release
+
+jobs:
+  build:
+    name: tag release
```

```suggestion
    name: generate-release
```
?

---

how about

```suggestion
    name: create draft release
```

why the dash? this is a name, not an ID, it can have spaces and is supposed to be human readable